### PR TITLE
Add hint for changed staging tests

### PR DESCRIPTION
**Pull request by OleksandrOrlov**
The PR adds the script that is executed on Travis run while PR is
created. The script checks if there are changes in test modules that are
used in staging test scenarios and puts a hint as a comment to the PR to
make a VR, confirming that the changes did not broke staging tests.

- Related ticket: https://progress.opensuse.org/issues/54311

### CODE REVIEW

**Comment by OleksandrOrlov** [comment]
Test

File: comment_on_staging_test_change.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#! /bin/sh
+
+# Run the script only in case if it is a PR
+if [ ! -z "$TRAVIS_PULL_REQUEST" ]; then
```
Comment (okurz) [inline comment_on_staging_test_change.sh]: I suggest to use an early return instead, e.g. `if [ -z "$TRAVIS_PULL_REQUEST" ] && exit 1`

File: comment_on_staging_test_change.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#! /bin/sh
```
Comment (okurz) [inline comment_on_staging_test_change.sh]: better use `#!/bin/sh -e` to not stumble over unhandled, silent errors :)

Also it seems like you are actually using bash syntax, e.g. for arrays, not POSIX sh

**Comment by OleksandrOrlov** [comment]
Closing the PR, as according to [Travis documentation](https://docs.travis-ci.com/user/environment-variables/):

> Encrypted environment variables are not available to pull requests from forks.

Another solution will be provided.

**Pull request closed without merging**